### DISCUSSION TITLE
Autograding Tests are always failing because the Setup step is not executed by the autograder

### DISCUSSION THREAD

**Discussion by samad-irisdata-pt** [community/community#113384] [GitHub Education]
### Hi everyone,

No test can be run in the autograding github platform because the setup phase is never executed. The setup phase is responsible for compiling and generate the executable. This phase is nor executing and the test has no executable to run.
This is a situation for some days.

**Comment by samus-aran** [comment]
Hi @samad-irisdata-pt 👋 

This seems to be a similar topic to the following discussion(s):

- https://github.com/orgs/community/discussions/106545

Please feel free to share your experiences and comments there. 👍  

We're closing this discussion because it appears to be a duplicate. Please re-open if you felt this was closed prematurely. Thank you for helping us maintain a productive and tidy community for all our members.
